xbmc/favourites/GUIWindowFavourites.cpp

```diff
@@ -117,6 +118,25 @@ class CVideoSelectActionProcessor : public VIDEO::GUILIB::CVideoSelectActionProc
     return true;
   }
 };
+
+class CVideoPlayActionProcessor : public VIDEO::GUILIB::CVideoPlayActionProcessorBase
```

Comment:
I'll have to implement something like this for discs too

Reply:
Good point. I was already of the opinion that I may have missed some places where we also kick off video playback action. :-) Maybe we can do this in a follow-up PR?

Reply:
`CAutorun::RunDisc` it is maybe.

Reply:
Yeah but the autorun itself also doesn't make sense (it's a poll approach from the CApp itself)...
Long story short, don't worry about this.

Reply:
> Long story short, don't worry about this.

 With pleasure. Had a quick look at the code and it felt like another can of worms. I think for Omega I already have refactored enough old code.